src/_cffi_src/openssl/ssl.py

```diff
@@ -296,6 +296,9 @@
 void SSL_CTX_set_info_callback(SSL_CTX *, void (*)(const SSL *, int, int));
 void (*SSL_CTX_get_info_callback(SSL_CTX *))(const SSL *, int, int);
 
+void SSL_CTX_set_msg_callback(SSL_CTX *, void (*)(int, int, int, const void *, size_t, SSL *, void *));
```

Comment:
You'll need to wrap this line due to length.

Reply:
Is there any linter/formatter that I should follow? Or is there any rules that I should follow?

Reply:
`tox -e flake` will run everything, but in short: we use black to auto-format, and ruff to enforce some other rules.

Reply:
I has having problem with building cryptography locally and couldn't run `tox`. Hope everything is good now.